pkg/envtest/webhook.go

```diff
@@ -429,3 +435,25 @@ func runtimeListToUnstructured(l []runtime.Object) []*unstructured.Unstructured
 	}
 	return res
 }
+
+// InstallWithCustomTLS installs specified webhooks to the API server but
+// doesn't setup a CA/cert via tinyca
+func (o *WebhookInstallOptions) InstallWithCustomTLS(config *rest.Config) error {
```

Comment:
This should not be fully duplicated from `Install()`. Either one should call the other or something, but a copy-paste is going to fall out of sync eventually.